rules/windows/initial_access_suspicious_ms_exchange_worker_child_process.toml

```diff
@@ -11,7 +11,7 @@ Identifies suspicious processes being spawned by the Microsoft Exchange Server w
 indicate exploitation activity or access to an existing web shell backdoor.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.process-*", "winlogbeat-*", "logs-windows.sysmon_operational-*", "endgame-*"]
+index = ["logs-endpoint.events.process-*", "winlogbeat-*", "logs-windows.sysmon_operational-*", "endgame-*", "logs-m365_defender.event-*", "logs-sentinel_one_cloud_funnel.*"]
```

Comment:
Btw it might make sense to space out this list for better readability, for rules where the index pattern list is this long. Not necessary though.